src/smc-webapp/customize.tsx

```diff
@@ -0,0 +1,270 @@
+/*
+ * decaffeinate suggestions:
```

Comment:
I think you fixed all these problems (I'm not seeing them here), so delete this decaf comment?